### PR TITLE
WIP: EOS-20722: Remove hardcoded install_path from build scripts

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
Signed-off-by: Rahul Tripathi <rahul.tripathi@seagate.com>

# Problem Statement
- install path needed to be removed from build script. Moved it to the top as a configurable variable.

# Design
-  NA

# Coding 
-  Coding conventions are followed and code is consistent [Y]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/setup.py [hunk 1]
```diff
@@ -19,6 +19,10 @@
 from typing import List
 from setuptools import setup
 
+# Set install path and related directories
+install_path = "/opt/seagate"
```
Comment (sachinpunadikar) [inline py-utils/setup.py:23]: Read it from cortx.conf.sample file

File: py-utils/utils-post-install [hunk 2]
```diff
@@ -14,10 +14,18 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+# Install_path
+install_path="/opt/seagate"
```
Comment (sachinpunadikar) [inline py-utils/utils-post-install:18]: read from cortx.conf.sample file

File: py-utils/utils-post-install [hunk 3]
```diff
@@ -14,10 +14,18 @@
...
-/bin/mkdir -p /opt/seagate/cortx/utils/bin
-/bin/cp -f /usr/lib/python3.6/site-packages/cortx/setup/utils_setup.py /opt/seagate/cortx/utils/bin/utils_setup
-/bin/chmod +x /opt/seagate/cortx/utils/bin/utils_setup
+/bin/mkdir -p $install_path/cortx/utils/bin
+/bin/cp -f /usr/lib/python3.6/site-packages/cortx/setup/utils_setup.py $install_path/cortx/utils/bin/utils_setup
+/bin/chmod +x $install_path/cortx/utils/bin/utils_setup
+
+# Copy cortx.conf.sample as cortx.conf
+if [ ! -f /etc/cortx/cortx.conf ]; then
+  cp ./cortx.conf.sample /etc/cortx/cortx.conf
```
Comment (sachinpunadikar) [inline py-utils/utils-post-install:30]: "./cortx.conf.sample" <- this is incorrect. You will get failure as file not found.

File: py-utils/setup.py [hunk 4]
```diff
@@ -87,9 +92,8 @@ def get_install_requirements() -> list:
       data_files = [ ('/var/lib/cortx/ha/specs', specs),
                      ('/var/lib/cortx/ha', ['src/utils/ha/hac/args.yaml',
                                             'src/utils/ha/hac/re_build.sh']),
-                     ('/opt/seagate/cortx/utils/conf',
-                          ['requirements.txt', 'src/setup/setup.yaml']),
-                     ('/opt/seagate/cortx/utils/conf', tmpl_files)],
+                     (conf_dir, ['requirements.txt', 'src/setup/setup.yaml']),
+                     (conf_dir, tmpl_files)],
```
Comment (sachinpunadikar) [inline py-utils/setup.py:96]: You missed to copy the cortx.conf.sample file. This file will be referred after installation

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please work on the comments

**Comment by rahul-tripathi-git** [comment]
Duplicate PR

**Pull request closed without merging**